src/style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
+  width: 300px;
+  height: 100px;
+  margin: auto;
+}
+
+#header {
+  background-color: blue;
+}
+
+#header:hover {
+  height: 200px;
+}
+
+#footer {
+  background-color: black;
+}
+
+#footer:hover {
+  height: 200px;
+}
+
+.important {
+  background-color: red;
+}
+
+.important:hover {
+  height: 200px;
+}
```

Comment:
you should write all hover styles using comma combiner
```
.important:hover,
#header:hover, {
```

Reply:
Ok, thank you)